### DISCUSSION TITLE
How to use Noisy Latent Composition?

### DISCUSSION THREAD

**Discussion by joywb** [comfyanonymous/ComfyUI#385] [Q&A]
https://comfyanonymous.github.io/ComfyUI_examples/noisy_latent_composition/

I couldn't find a workflow diagram in the example. Can anybody please tell me where I can obtain or learn about this operation?

**Comment by playlogitech** [comment]
download any picture from the link you sent and import into comfy ui like a preset

**Reply by joywb** [reply]
thanks! It's work!

**Comment by joywb** [comment]
Done

**Comment by yuragorlo** [comment]
I can't understand how it works.
pictures from the link in png format, but uploading preset files required  JSON or CSV formats.
Can you help me?

**Reply by mcmonkey4eva** [reply]
the pictures are comfy workflows, not swarm presets

**Comment by yuragorlo** [comment]
Hello!
thank you for trying to help me: I just started doing this and I'm a bit dumb)
I understand that you can upload a workflow picture, and then upload the missing weights and it will work.
But the picture has to contain the workflow! At the link above, the author posted only the results of the generation - a PNG file with people on it. And I don't understand how you can recreate and then run this from the resulting png only. I think that it is not possible and you need either a JSON file or a picture with the workflow itself, not the result of this workflow. Where am I wrong?

**Reply by mcmonkey4eva** [reply]
just drag a comfy image into the Comfy Workflow page and it will load the workflow from the image

**Comment by yuragorlo** [comment]
WOW! Thank you so much! It works!
